### PR TITLE
Add table forms for expose validation redirect report

### PR DESCRIPTION
**Pull request by awmackowiak**
Screenshot 2022-12-08 at 09 59 08

### CODE REVIEW

File: vaas/vaas/router/api.py [hunk 1]
```diff
@@ -270,8 +271,6 @@ def obj_get(self, bundle, **kwargs):
 
     def get_object_list(self, request):
         return None
-
-
```
Comment (ziollek) [inline vaas/vaas/router/api.py:274]: Does linter accept that formatting?

File: vaas/vaas/router/api.py [hunk 2]
```diff
@@ -313,13 +309,13 @@ def obj_create(self, bundle, **kwargs):
         bundle = self.full_hydrate(bundle)
         task = fetch_redirects_async.apply_async(task_id=bundle.obj.pk)
         bundle.obj.status = task.status
-        bundle.obj.output = task.result
+        bundle.obj.output =task.result
```
Comment (ziollek) [inline vaas/vaas/router/api.py:312]: ?

File: vaas/vaas/router/forms.py [hunk 3]
```diff
@@ -72,7 +71,7 @@ class Meta:
                 choices=tuple((action.action, action.name) for action in configuration.actions)
             ),
             'priority': PrioritySelect(
-                choices=tuple([(i, i) for i in range(1, 500)]),
+                choices=tuple([(i, i) if i != 0 else ('', '') for i in range(0, 500)]),
```
Comment (ziollek) [inline vaas/vaas/router/forms.py]: What is the reason for that change?

**Review by ziollek — CHANGES_REQUESTED** [review]

File: vaas/vaas/router/api.py (hunk 1 above)
Comment (awmackowiak) [reply vaas/vaas/router/api.py]: Yes. Something goes wrong.

File: vaas/vaas/router/forms.py (hunk 3 above)
Comment (awmackowiak) [reply vaas/vaas/router/forms.py]: This was quick proposition of fixing priority calculation in admin forms. @smigacz will do it later.
I remove this solution in this PR.

File: vaas/vaas/router/api.py (hunk 2 above)
Comment (awmackowiak) [reply vaas/vaas/router/api.py]: It has been fixed.

**Review by ziollek — APPROVED** [review]

**Review by ziollek — APPROVED** [review]

**Pull request merged**